osidb/serializer.py

```diff
@@ -366,6 +367,35 @@ def update(self, instance, validated_data):
         validated_data = self.embargoed2acls(validated_data)
         return super().update(instance, validated_data)
 
+    def validate(self, data):
+        """
+        validate that the current user is member of all LDAP groups (s)he provides
+        with the access so there is no access expansion beyond the user permissions
+        """
+        data = super().validate(data)
+
+        # get the resulting ACLs from the embargo status
+        embargoed = self.context["request"].data.get("embargoed")
```

Comment:
if you manually instantiate this serializer (because of a custom view or something like that) then `request` won't be included in the context by default, DRF does it when instantiating it but this can easily turn into a mess, I'm not sure if we can have a serializer-only field that accepts both read and write operations, but worth a look maybe

Reply:
I would just document it for now, it's nothing too serious and I have no better suggestions for now, I'd suggest putting a comment/warning somewhere and having a JIRA ticket for tracking this as an improvement